bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -565,6 +565,13 @@
             name="%view.name"
             restorable="true">
       </view>
+      <view
+            class="com.espressif.idf.ui.dialogs.HintsView"
+            icon="icons/Hints_View.png"
+            id="com.espressif.idf.ui.views.hints"
+            name="%hints_view.name"
+            restorable="true">
+      </view>
```

Comment:
Could you create an Espressif category for views and add Hints as part of it. I think Ali added Callers View previously, we could put these two items as part of the Espressif view. 

This would help users to find it easily

Reply:
done